lib/loader.js

```diff
@@ -16,13 +16,17 @@ module.exports = function(content, map) {
 	var root = query.root;
 	var moduleMode = query.modules || query.module;
 	var camelCaseKeys = query.camelCase || query.camelcase;
+	var sourceMap = query.sourceMap || false;
 	var resolve = createResolver(query.alias);
 
-	if(map !== null && typeof map !== "string") {
+	if(sourceMap && map !== null && typeof map !== "string") {
 		map = JSON.stringify(map);
+	} else {
+		map = null;
```

Comment:
@bebraw yep, example `postcss-loader` now always generate source map and send their, we should remove their